compiler/testData/diagnostics/tests/variance/InvariantPosition.fir.kt

```diff
@@ -38,10 +38,10 @@ interface Test<in I, out O, P> {
     var neOk22: Inv<out O>
     var neOk23: Inv<out I>
 
-    var neOk30: <!WRONG_NUMBER_OF_TYPE_ARGUMENTS!>Pair<I, ><!>
+    var neOk30: <!WRONG_NUMBER_OF_TYPE_ARGUMENTS, WRONG_SETTER_PARAMETER_TYPE!>Pair<I, ><!>
```

Comment:
All changes in this file look not correct: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/variance/InvariantPosition.kt#L41

Btw, these properties don't have user-contributed setters at all, which means you need to bail out for default accessors.